concurrency/pkg/reconciler/concurrency/concurrencycontrol.go

```diff
@@ -127,11 +133,17 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, pr *v1beta1.PipelineRun)
 	return r.updateLabelsAndStartPipelineRun(ctx, pr)
 }
 
-func matches(pr *v1beta1.PipelineRun, cc *v1alpha1.ConcurrencyControl) bool {
+// Matches returns true if the PipelineRun is selected by the ConcurrencyControl's selector.
+// An empty selector always matches the PipelineRun.
+func Matches(pr *v1beta1.PipelineRun, cc *v1alpha1.ConcurrencyControl) bool {
```

Comment:
why do you export `Matches`?

Reply:
In this PR, primarily to make it easier to test. However, I think it's useful to know whether a concurrency control applies to a given pipelinerun (i.e. whether the control matches the pipelinerun) and would be a useful function to export.